### ISSUE TITLE
Empty &cc= and &bcc= params are sent for default app if there are no values for it

### ISSUE THREAD

**Issue by stefandries** [mariordev/mailtoui#11]
My mailto: does not contain a CC or BCC param, but still the URL called for the default mail app contains the (empty) params. This causes issues with desktop version of Outlook, as it puts "&bcc=" in the CC: field.

My code: 

`<a class="maiiltoui" href="mailto:test@mail.com&subject=Demo&body=Great piece of code, Mario!">click here</a>`

![image](https://user-images.githubusercontent.com/39307740/64694435-f2cb3800-d499-11e9-87bc-400399e42106.png)

**Comment by mariordev** [comment]
Yeah, sadly Outlook doesn't support cc or bcc from a mailto link. My guess is that this bug has to do with that. I'll take a look as soon as I can. Thanks for reporting it, @stefandries !
